x-pack/plugins/session_view/public/components/DetailPanelAccordion/styles.ts

```diff
@@ -0,0 +1,37 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import { useMemo } from 'react';
+import { CSSObject } from '@emotion/react';
+
+export const useStyles = () => {
```

Comment:
Cool! We use styled components within most of our apps at the moment, but I'm not sure which is better. I would touch base with someone from our EUI team like @cchaos to see what their opinion on these are. I know they're looking to remove custom styling as much as possible. Just an FYI